src/NzbDrone.Core/Localization/Core/en.json

```
@@ -509,6 +509,7 @@
   "HardlinkCopyFiles": "Hardlink/Copy Files",
   "HasMissingSeason": "Has Missing Season",
   "Health": "Health",
+  "HealthMessagesInfoBox": "You can find more information about the cause of these health check messages by clicking the wiki link at the end of the row, or by checking your [logs]({link}). If you have difficultly interpreting these messages then you can reach out to our support, at the links below.",
```

```suggestion
  "HealthMessagesInfoBox": "You can find more information about the cause of these health check messages by clicking the wiki link (book icon) at the end of the row, or by checking your [logs]({link}). If you have difficulty interpreting these messages then you can reach out to our support, at the links below.",
```

should this also suggest looking at events as well?